muxrpc/test/go/roomstate_test.go

```diff
@@ -0,0 +1,158 @@
+package go_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.cryptoscope.co/muxrpc/v2"
+
+	"github.com/ssb-ngi-pointer/go-ssb-room/muxrpc/handlers/tunnel/server"
+)
+
+// peers not on the members list can't connect
+func TestStaleMembers(t *testing.T) {
+	testInit(t)
+
+	r := require.New(t)
+
+	testPath := filepath.Join("testrun", t.Name())
+	os.RemoveAll(testPath)
+
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
+	t.Cleanup(cancel)
+
+	// create the roomsrv
+	ts := makeNamedTestBot(t, "server", ctx, nil)
+	ctx = ts.ctx
+
+	// two new clients, connected to server automaticaly
+	// default to member during makeNamedTestBot
+	tal := ts.makeTestClient("tal")
+	srh := ts.makeTestClient("srh")
```

Comment:
wtf is srh and tal

Reply:
oh shit. it's just names. i was bored with alice and bob.

Reply:
haha oh okay ^^

Reply:
alice and bob are out, tal and srh are IN

Reply:
https://en.wikipedia.org/wiki/Sarah
https://en.wikipedia.org/wiki/Tal_(name)